crypto/rand/rand_lib.c

```diff
@@ -113,7 +216,24 @@ void rand_cleanup_int(void)
     CRYPTO_THREAD_lock_free(rand_engine_lock);
 #endif
     CRYPTO_THREAD_lock_free(rand_meth_lock);
-    rand_drbg_cleanup();
+    CRYPTO_THREAD_lock_free(rand_bytes.lock);
+    OPENSSL_clear_free(rand_drbg.randomness, rand_drbg.size);
```

Comment:
Thd DRBG clean-entropy callback is called as soon as the entropy is consumed, so I think it's not needed.  But for parallel construction I did it anyway in the new PR.